app/views/admin/loans/timeline/_step.slim

```diff
@@ -1,15 +1,41 @@
-- wrapper_tag = {tag: is_first ? 'div' : 'tr'}
+- wrapper_tag = {tag: is_first ? "div" : "tr"}
+
+- draft = entry.is_finalized ? "" : "draft"
+- completed = entry.completed? ? "completed" : ""
+- classes = "#{completed} #{draft}"
 
 *wrapper_tag
   / Add spacer for step who has a group as a sibling
   - unless siblings_are_all_steps
     td.spacer colspan="#{max_group_depth - entry.depth + 1}"
-  td = entry.summary
-  td = entry.display_start_date ? l(entry.display_start_date, format: :short) : ''
-  td = entry.display_end_date ? l(entry.display_end_date, format: :short) : ''
-  td
+
+  td class="#{classes}"
+    input.select-step type="checkbox" class=(entry.completed? ? "completed-item" : "incomplete-item") data-id=entry.id
```

Comment:
I'm not sure why we have this here unless it is here for when we select all complete/incomplete items. But that would not be the best way to go about selecting all completed, given that completed class will be in the parent element. Just something to consider.